meta-ov/recipes-support/autofs-config/files/boot.mount

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Mount the boot partition
+
+[Mount]
+What=/dev/mmcblk0p1
+Where=/boot
```

Comment:
Would `/etc/fstab` entry be a simpler solution for this?

Reply:
Think we can change this afterwards as well, but it doesn't matter from my point of view ..